docs/variable.properties

```diff
@@ -1,2 +1,2 @@
 current_version=0.2.10
-worker_sdk_version=14.0
\ No newline at end of file
+worker_sdk_version=14.2
```

Comment:
should it be 14.2.1?

Reply:
there's no docs versions for patch releases